docker/daemon_unix_test.go

```diff
@@ -0,0 +1,42 @@
+// +build daemon,!windows
+
+package main
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/crosbymichael/docker/daemon"
```

Comment:
lol that's some funny `goimports`, it's because `crosbymichael` is before `docker` lexicographically :D